src/js/order.js

```diff
@@ -652,19 +651,22 @@ function fetchYear(year, request_scheduler, nocache_top_level) {
             '&unifiedOrders=1' +
             '&returnTo=' +
             '&orderFilter=year-%(year)s' +
-            '&startIndex=%(startOrderPos)s'],
+            '&startIndex=%(startOrderPos)s' +
+            '&language=en_GB'],
```

Comment:
These added language codes need to go - same reason as last time - I'll be sensible and only write one comment rather than foam at the mouth.

Reply:
Sorry, this shouldn't be here. I must have mixed two branches. The new PR (I deleted my copy of the branch) should fix this.